spec/base_decorativeb.rb

```diff
@@ -0,0 +1,19 @@
+require 'rspec'
```

Comment:
- [ ] Please keep in mind that the test files should end with `_spec` so that when running `rspec ./spec/` command line, it knows what the files contain the tests to be run. 🙏 I had to run each file in a row as shown in the screenshots I uploaded to run the tests. 👍